index.js

```diff
@@ -7,4 +13,62 @@ const fgMultipleHooks = async (req, res, ...multipleHooks) => {
     return shouldAbort;
 };
 
-module.exports = fgMultipleHooks;
\ No newline at end of file
+const defaultLastOnResponse = async (req, res, stream) => {
```

Comment:
the name "default" does not give a clear indication of what this hook does, maybe afterResponseSendStreamUntouched or something less verbose but with more meaning